### ISSUE TITLE
Settings "modified" text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52580]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Windows 10

Additional Details:
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1.Launch VS Code 
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown-->New settings editor-->Show Modified only checkbox and mark it as check-->Workbench-->Modified(in green color)
3)Use color contrast analyzer.

**Actual**
Color contrast ratio is less than 4.5:1 for the content present on the page. i.e. Modified(in green color)

**Expected**
Color contrast ratio should be more than 4.5:1 for the content present on the page. i.e. Modified(in green color)

**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact**:
The End user will not be able to view the content properly.

**MAS Reference**:
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment For Reference:
![accessibility a11y_vscode_previewsettings_colorcontrast](https://user-images.githubusercontent.com/39204909/41713059-be1a395c-7569-11e8-883a-31139b85f830.png)

**Comment by misolori** [comment]
This only fails on our dark theme

**Comment by AccessibilityTestingTeam-TCS** [comment]
The bug is verified on the below given build:
version: 1.27.0-insider
The bug is fixed.
